### ISSUE TITLE
The Chrome DevTools Accessibility page has lots of things missing

### ISSUE THREAD

**Issue by slauriat** [GoogleChrome/developer.chrome.com#3459]
I've discovered a lot of accessibility features and many of them don't appear on the [Chrome DevTools Accessibility page](https://developer.chrome.com/docs/devtools/accessibility/reference/). How can I help/support revamping it to cover the features more cohesively & comprehensively?

- Inspector
  - Source order viewer: shows the source order of the children of the selected container
- Settings
  - Preferences
    - Color format
    - Show user agent shadow DOM
- chrome://accessibility for the full accessibility tree information
- Rendering tools
  - Emulate a focused page
  - Enable automatic dark mode
  - Emulate CSS media feature prefers-color-scheme
  - Emulate CSS media feature forced-colors (= Windows high contrast mode)
  - Emulate CSS media feature prefers-contrast
  - Emulate CSS media feature prefers-reduced-motion
  - Emulate vision deficiencies
- Device toolbar
  - Responsive design test for [WCAG Reflow](https://www.w3.org/TR/WCAG21/#reflow): uses mobile user agent, don't use the built-in one! Add custom device, 320 wide, desktop

**Comment by jecfish** [comment]
Hi @slauriat,

Our technical writer is working on updating DevTools documentation. However,  we would appreciate your help on this too!

On top of my head, these are some help we need:

Documentation we already have, but we might want to link it somehow in a11y section:
- A **new docs on how to fix low contrast text** (Part of #3266). We have a video already, but no relevant docs. 
  - Most of the rendering tools you mentioned above fits in this article, plus some other features. 
  - We recently added [Rendering tools](https://developer.chrome.com/docs/devtools/rendering/emulate-css/) docs + videos. Instead of explaining the tools again under a11y, I suggest we have a section explaining the a11y usage of this features (if any), then link them to this article as much as possible.
  - On top of that: [Automatic low contrast report](https://developer.chrome.com/blog/new-in-devtools-90/#low-contrast) and [APCA contrast calculation](https://developer.chrome.com/blog/new-in-devtools-89/#apca) should be mentioned in this docs as well.
- Add a **new section for Source order viewer** under existing [Accessibility features reference](https://developer.chrome.com/docs/devtools/accessibility/reference/).
- Add a **new section for Settings** under existing [Accessibility features reference](https://developer.chrome.com/docs/devtools/accessibility/reference/). If this feels like fit into the "fix low contrast text" docs, include it there instead.
- Add a **new section for Device toolbar** under existing [Accessibility features reference](https://developer.chrome.com/docs/devtools/accessibility/reference/).
- Add a **new section for Accessibility tree** to include both **chrome://accessibility** and move the existing [Full-page accessibility tree](https://developer.chrome.com/docs/devtools/accessibility/reference/#explore-tree) under it. 
- Update the existing [Accessibility pane](https://developer.chrome.com/docs/devtools/accessibility/reference/#pane) section to mention the [pane can be dragged to the front](https://developer.chrome.com/blog/new-in-devtools-103/#reorder-pane) for easier access.

Each point deserve a separate PR. Smaller PR helps us to review the effectively. Let me know if you interested in extending your help for any of these. You don't need to write all of that of course. :)

**Comment by dstmarthe** [comment]
Thanks for raising this. We’ve moved to a new platform, and a bunch has changed. @slauriat while there isn't an improve button anymore - I’ve moved the issue to our new issue tracker and you can continue the conversation here: https://issuetracker.google.com/issues/324330166
